test_helpers/tcs/tcs.go

```diff
@@ -0,0 +1,98 @@
+package tcs
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/tarantool/go-tarantool/v2"
+	"github.com/tarantool/go-tarantool/v2/test_helpers"
+)
+
+type Tcs struct {
+	// conn Connection interface for interacting with Tarantool.
+	doer tarantool.Doer
+}
+
+// GetResponse is a response of a get request.
+type GetResponse struct {
+	Data []struct {
+		Path        string
+		Value       string
+		ModRevision int64 `mapstructure:"mod_revision"`
+	}
+}
+
+// StartTcs starts a Tarantool centralized configuration storage.
+// Returns a Tcs instance and a cleanup function.
+func StartTcs(opts test_helpers.StartOpts) (Tcs, func()) {
+	if less, err := test_helpers.IsTarantoolEEVersionLess(3, 3, 0); less || err != nil {
+		log.Fatal("Tarantool EE 3.3+ required", err)
+	}
+	need_clean := opts.WorkDir == ""
+
+	inst, err := test_helpers.StartTarantool(opts)
+	if err != nil {
+		log.Fatalf("Failed to start Tarantool config instance: %v", err)
+	}
```

Comment:
No, let's do the following here:

1. Create a temporary directory.
2. Write a TCS config into the directory .
3. Start TCS with the config and workdir.

Now this will have to be done in the test code on the user side, which is not very convenient.

It would be a good idea to implement Endpoints() methods for TCS too. It would be to select ports for the test cluster from free ports. But it would be ok for the first implementation to use some pre-defined ports here.

Reply:
The idea here is to do all this machinery for the user and provide some default implementation that will work in 99% of cases. The user just needs to run TCS and get endpoints+credentials to connect.

If the user wants to create folders and copy something himself, he will be able to do it this with a raw StartTarantool().